### ISSUE TITLE
Missing cargo package

### ISSUE THREAD

**Issue by wolfiestyle** [mrDIMAS/rusty-editor#7]
The only distribution method provided at the moment seems to be an `.exe` file for windows. As a linux user I would want to do `cargo install rusty-editor` but there is no package published for it.

**Comment by mrDIMAS** [comment]
Good catch. At first I should learn how to create such packages :thinking:

**Comment by wolfiestyle** [comment]
I think as long as it's marked as a binary package is fine. Also it needs to include the Cargo.lock

**Comment by phenax** [comment]
+1 for this. You can publish this on crates -> https://doc.rust-lang.org/cargo/reference/publishing.html
Also, is the `*.lock` in `.gitignore` intentional? I think that should be removed.
